app/models/subscription.rb

```diff
@@ -4,8 +4,14 @@ class Subscription < ActiveRecord::Base
   validates :group, presence: true
   validates_inclusion_of :kind, in: ['trial', 'gift', 'paid']
 
+  delegate :id, to: :group, prefix: true, allow_nil: true
```

Comment:
We don't usually create subscriptions independently of groups, so this is awkward (we're basically defining a wannabe foreign key for when we create a subscription so we can define the group... making me wonder if we have this `has_one`/`belongs_to` relationships backwards?)

Also I just wanted to get this out the door; let me know if I'm missing something obvious @robguthrie